### PR TITLE
Provide a better error message when invoking a non-existant hub method

### PR DESCRIPTION
**Pull request by BrennanConroy**
#2313

### CODE REVIEW

**Comment by BrennanConroy** [comment]
I'll add a test for invoking a non-existant hub with arguments

File: src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs [hunk 1]
```diff
@@ -136,7 +136,7 @@ public override IReadOnlyList<Type> GetParameterTypes(string methodName)
         {
             if (!_methods.TryGetValue(methodName, out var descriptor))
             {
-                return Type.EmptyTypes;
+                throw new HubException("Method doesn't exist.");
```
Comment (mikaelm12) [inline src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs]: Include the name of the method that doesn't exist

Comment (anurse) [inline src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs]: Let's do "does not"

Comment (anurse) [reply src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs]: It's already in the message that gets generated. See the tests.

Comment (mikaelm12) [reply src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs]: Derp

**Pull request merged**